src/patternfly/layouts/Page/page.scss

```diff
@@ -135,6 +136,13 @@
   .pf-l-page__header-tools {
     padding-top: var(--pf-l-page__header-tools--MarginTop);
     padding-bottom: var(--pf-l-page__header-tools--MarginBottom);
+
+    .pf-c-dropdown__toggle-text {
+      max-width: var(--pf-l-page__header-tools-c-dropdown__toggle-text--MaxWidth);
```

Comment:
```suggestion
      max-width: var(--pf-l-page__header-tools--c-dropdown__toggle-text--MaxWidth);
```

Reply:
Oh and should the dropdown's parent define the max-width of the dropdown component? I thought that's what we discussed on Wednesday. I don't think it matters.

Reply:
ya I'm not sure, I only got as far as it wouldn't be defined in the component itself.